packages/browser-extension/src/_locales/en/messages.json

```diff
@@ -62,6 +62,9 @@
   "reportDialog_popupShowUpOption": {
     "message": "Popup showed up"
   },
+  "reportDialog_ScrollingIssues": {
```

Comment:
Please, the key would be `reportDialog_cannotScrollOption` so update the rest of keys in consequence to this change.

Reply:
Ok